docs/ReleaseNotes.md

```diff
@@ -1,6 +1,66 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.1
+Acuminator 2.1 includes the diagnostics, code fixes, enhancements, and bug fixes described in this section, as well as the features that have been implemented in previous versions.
+
+### New Diagnostic and Code Fixes
+In this version, the diagnostic and code fixes have been added for the following issue.
+
+| Code                            | Issue Description                                                | Type    | Diagnostics | Code Fix    | 
+| ------------------------------- | ---------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1007](diagnostics/PX1007.md) | A public class should have a description in the `summary` XML tag. | Warning (Level 3: Informational)	 | Available | Available   |
```

Comment:
@k-s-popova  the default diagnostic severity is currently a "Warning", not "Informational". Should we changed that? But informational severity will most likely pass our regress tests and overall is hard to notice in the Visual Studio

Reply:
@SENya1990 , so which type it has? 
We have three types of warnings (besides errors and messages) for ISV certification: 
Warning (Level 1: Significant)
Warning (Level 2: Production Quality)
Warning (Level 3: Informational)

Reply:
Well, I'm not sure about ISV levels. I was referring to VS IDE Severity levels. I'm not sure how they are converted to ISV warnings.
@dropsonic  may be you can help?

Reply:
@SENya1990 ISV levels are used for information purposes. They do not exist in Visual Studio.
Dmitriy Naumov can help with determining the corresponding level because he is the main stakeholder for these levels.

Reply:
Well I just wanted to say that there is no mapping between the IDE severity and ISV protection level and the developer will see the IDE severity. I'm not sure if many developers even know about ISV protection levels at least the devs not working for ISV. So I'm not sure that we should write them in the documentation for the diagnostic - it can be confusing for other devs. At least I would suggest to write both severity and ISV protection level, or discuss the mapping between severity and ISV protection level, like IDE severity Warning should be a "Warning" protection level

Reply:
Because it will be confusing for me to find a "Warning" from PX1007 in the IDE, then open the documentation and see an "Information" protection level. There are devs who strive to have no warnings at all and the difference between "Warning" and "Information" is big for them.

Reply:
It exists since the beginning. There is a clear mapping between ISV severity and VS severity: Error in VS is Error for ISV; Warning in VS is Warning for ISV, but there is 3 levels of warnings for ISV; Information is Information too. So it is impossible to have Information in the docs and Warning in VS. It is just a more granular severity description for ISVs.
It was created for ISV Certification Program by Ruslan to provide clear requirements for different certification levels.

Reply:
I renamed Level to ISV Level so that it is clear that these levels are only for ISVs.

Reply:
Yes, I just wanted for the text to more clearly indicate that it is related to ISV. The current text seems fine to me